tools/ci_static_check_reports/build_static_checkers_reports.py

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+#
+import os
+import sys
+from pathlib import Path
+from typing import Dict, List
+
+from invoke import Context
+
+sys.path.insert(0, "airbyte-integrations/connectors")
+from tasks import CONFIG_FILE, TOOLS_VERSIONS, _run_task  # noqa
+
+TASK_COMMANDS: Dict[str, List[str]] = {
+    "black": [
+        f"pip install black~={TOOLS_VERSIONS['black']}",
+        f"XDG_CACHE_HOME={os.devnull} black -v {{check_option}} --diff {{source_path}}/. > {{reports_path}}/black.txt",
+    ],
```

Comment:
Why are double squiggle brackets used here? other lines have single ones only.

Reply:
we need this to keep param to later format it. E.g `f"XDG_CACHE_HOME={os.devnull} black -v {{check_option}} --diff {{source_path}}/. > {{reports_path}}/black.txt"` becomes first `"XDG_CACHE_HOME=/dev/null black -v {check_option} --diff {source_path}/. > {reports_path}/black.txt"` to be later formatted one more time it `tasks.py`.